partner_changeset/README.rst

```diff
@@ -0,0 +1,96 @@
+==================
```

Comment:
Hi @elicoidal 
Thanks for your eager review :-)
I just extracted this addon from a project's branch and am currently adding the OCA's things, that's why I put `[WIP]` in the PR's subject.